test/journal/update_journal_entry_test.js

```diff
@@ -105,6 +106,26 @@ describe("Journal", function () {
                 expect(response.body.mood).to.equal("");
             });
         });
+        //moodEmoji
+        it("rejects a blank mood Emoji", function () {
+            return updatePatientEntry({}, {
+                moodEmoji : ""
+            }).then(function (response) {
+                expect(response).to.be.an.api.error(400, "invalid_emoji");
+                expect(response.body.moodEmoji).to.equal("");
+            });
+        });
+        it("allows a null mood Emoji", function (){
+            return updatePatientEntry({}, {
+                moodEmoji: null
+            }).then(function (response) {
+                console.log(response.body.errors);
```

Comment:
Remember to remove the `console.log`s before committing. No big deal for individual commits, but try to remove before a PR.